AsyncSwift/Views/SessionView.swift

```diff
@@ -36,50 +35,60 @@ struct SessionView: View {
 private extension SessionView {
 
     var sessionDetail: some View {
-        VStack(alignment: .leading, spacing: 0) {
-            Text(session.title)
-                .font(.title3)
-                .fontWeight(.semibold)
-                .padding(.vertical, 24)
-            VStack(alignment: .leading, spacing: 8) {
-                ForEach(session.description, id: \.self) { paragraph in
-                    Text(paragraph.content)
+        HStack(spacing: 0) {
+            VStack(alignment: .leading, spacing: 0) {
+                Text(observed.session.title)
+                    .font(.title3)
+                    .fontWeight(.semibold)
+                    .padding(.vertical, 24)
+                VStack(alignment: .leading, spacing: 8) {
```

Comment:
```suggestion
                VStack(alignment: .leading) {
```

8이 default value였던 것 같아서 생략 가능할까요 ?